_monitoring-plugins/alerting/monitors.md

```diff
@@ -44,6 +32,28 @@ Tag | A label that can be applied to multiple queries to combine them with the l
 Alert | An event associated with a trigger. When an alert is created, the trigger performs *actions*, which can include sending a notification.
 Action | The information that you want the monitor to send out after being triggered. Actions have a *destination*, a message subject, and a message body.
 Destination | A reusable location for an action. Supported locations are Amazon Chime, Email, Slack, or custom webhook.
+Finding | An entry for an individual document found by a per document monitor query that contains the Document ID, index name, and timestamp. Findings are stored in the Findings index: `.opensearch-alerting-finding*`.
+
+## Per document monitors
+
+Introduced 2.0
+{: .label .label-purple }
+
+The per query and per bucket monitors can only take a single query with one trigger condition. Per document monitors allow you to combine multiple query trigger conditions by adding a tag to the queries. Then you can add the tag as a single trigger condition instead of specifying a single query. The Alerting plugin processes the trigger conditions from all queries as a logical OR operation, so if any of the query conditions are met, it generates the alert notification.
```

Comment:
More of a nit: 

```suggestion
The per query and per bucket monitors can only run a single query with one trigger condition. Per document monitors allow you to combine multiple query trigger conditions by adding a tag to the queries. Then you can add the tag as a single trigger condition instead of specifying a single query. The Alerting plugin processes the trigger conditions from all queries as a logical OR operation, so if any of the query conditions are met, it generates the alert notification.
```

Reply:
Also, with Notifications and Alerting plugin merging together, do we know the distinction between which monitors run through the Notifications plugins vs the Alerting plugin? Not sure we can account for this in this PR because this appears in the "Alerting" section, but we probably should seek some more clarity on how the Notifications and Alerting plugins interact together.

@keithhc2, you probably know more about this than I do. Thoughts?

Reply:
I wouldn't say they "merge" together. It's more like Alerting runs a monitor, the monitor triggers an alert, Alerting lets Notifications know, and Notifications sends out a notification to a channel. So, I would say they work together, but I don't think we need to merge content in any way.